modules/mcuboot/CONFIG_BUILD_WITH_TFM.conf

```diff
@@ -0,0 +1,7 @@
+# This Kconfig fragment is applied to the MCUBoot configuration when
+# we are building for TF-M. This is necessary so MCUBoot can configure
+# itself to boot TF-M correctly.
+
+CONFIG_FPROTECT=n
+CONFIG_ARM_MPU=n
+CONFIG_MCUBOOT_CLEANUP_ARM_CORE=y
```

Comment:
Hi,

disabling ARM_MPU is sane because it is TF-M's responsibility to do access control. Not the bootloader.
Not disabling it causes crashes. See https://projecttools.nordicsemi.no/jira/browse/NCSDK-10736 for details.

I don't know why we need to cleanup the arm core or disable FPPROTECT.

But disabling FPPROTECT also sounds sane as that is a form of access control.

Reply:
Please add (parts of) these descriptions to the code itself in the form of comments.